### PR TITLE
[Core] templates cleaning

### PR DESCRIPTION
**Pull request by philbucher**
this is annoying to do and annoying to review, sorry for the inconvenience

I am a bit sick of pointing out the same things in review over and over again, this is the better solution :)

It is best to hide the whitespace changes

### CODE REVIEW

**Comment by jcotela** [comment]
@philbucher not sure if you are already doing this, but if not, could you please replace tabs by spaces on the headers of the templates? If I remember correctly, the "default kratos license" and "authors" lines use tabs in the current templates...

**Comment by philbucher** [comment]
@jcotela done
I hope at least :)

File: kratos/templates/classes/condition_template.cpp [hunk 1]
```diff
@@ -48,53 +46,58 @@ namespace Kratos
...
-@{KRATOS_NAME_CAMEL} & @{KRATOS_NAME_CAMEL}::operator=(@{KRATOS_NAME_CAMEL} const& rOther) {
-  BaseType::operator=(rOther);
-  Flags::operator =(rOther);
-  // mpProperties = rOther.mpProperties;
-  return *this;
+@{KRATOS_NAME_CAMEL} & @{KRATOS_NAME_CAMEL}::operator=(@{KRATOS_NAME_CAMEL} const& rOther)
+{
+    BaseType::operator=(rOther);
+    Flags::operator =(rOther);
+    // mpProperties = rOther.mpProperties;
```
Comment (pooyan-dadvand) [inline kratos/templates/classes/condition_template.cpp:99]: Why this is commented?

File: kratos/templates/classes/condition_template.cpp [hunk 2]
```diff
@@ -159,25 +165,27 @@ Condition::Pointer @{KRATOS_NAME_CAMEL}::Clone(IndexType NewId, NodesArrayType c
...
-  if (rResult.size() != number_of_nodes)
-    rResult.resize(number_of_nodes, false);
+void @{KRATOS_NAME_CAMEL}::EquationIdVector(EquationIdVectorType& rResult, ProcessInfo& CurrentProcessInfo)
+{
+    unsigned int number_of_nodes = GetGeometry().PointsNumber();
+    if (rResult.size() != number_of_nodes)
+        rResult.resize(number_of_nodes, false);
 
-@{KRATOS_CONDITION_ECUATION_ID_DOFS}
+    @{KRATOS_CONDITION_ECUATION_ID_DOFS}
```
Comment (pooyan-dadvand) [inline kratos/templates/classes/condition_template.cpp]: @roigcarlo there is a typo here...

File: kratos/templates/classes/element_template.cpp [hunk 3]
```diff
@@ -48,53 +46,60 @@ namespace Kratos
...
-@{KRATOS_NAME_CAMEL} & @{KRATOS_NAME_CAMEL}::operator=(@{KRATOS_NAME_CAMEL} const& rOther) {
-  BaseType::operator=(rOther);
-  Flags::operator =(rOther);
-  // mpProperties = rOther.mpProperties;
-  return *this;
+@{KRATOS_NAME_CAMEL} & @{KRATOS_NAME_CAMEL}::operator=(@{KRATOS_NAME_CAMEL} const& rOther)
+{
+    BaseType::operator=(rOther);
+    Flags::operator =(rOther);
+    // mpProperties = rOther.mpProperties;
```
Comment (pooyan-dadvand) [inline kratos/templates/classes/element_template.cpp:101]: again....

**Review by pooyan-dadvand — COMMENTED** [review]
Thanks @philbucher. Does it work with application generator of @roigcarlo ?

File: kratos/templates/classes/condition_template.cpp (hunk 2 above)
Comment (roigcarlo) [reply kratos/templates/classes/condition_template.cpp]: Fixed

**Comment by roigcarlo** [comment]
Afak changes should work as the generator works on tags, it doesn't care about spaces tabs etc...

File: kratos/templates/classes/condition_template.cpp (hunk 1 above)
Comment (philbucher) [reply kratos/templates/classes/condition_template.cpp:99]: no clue :)
should I uncomment?

File: kratos/templates/classes/element_template.cpp (hunk 3 above)
Comment (philbucher) [reply kratos/templates/classes/element_template.cpp:101]: same

**Comment by pooyan-dadvand** [comment]
So @roigcarlo please approve it if it works for you!

**Comment by roigcarlo** [comment]
I havn't test it, in my computer, but looks ok. Let me check it tomorrow in the office.

**Comment by philbucher** [comment]
have you tried @roigcarlo ?

**Comment by roigcarlo** [comment]
Ups sorry forgget about it. Give me a minute

**Review by roigcarlo — APPROVED** [review]
Yup all working fine. There are some tabs misplaced in the generated files, but really nothing important :+1:

**Comment by philbucher** [comment]
thanks!
if you tell me where / push it to this branch it could be fixed right away ... ?

**Comment by roigcarlo** [comment]
Mm here and there, but I suspect some of them were already there. I would merge as it is right now.

**Pull request merged**